components/fuipwahef749384h2e.yml

```diff
@@ -0,0 +1,23 @@
+name: emotional_bot_response
+display_name: Emotional Bot Response
+component_type: Generative
+model_type: NN-based
+is_customizable: true
+author: publisher@deeppavlov.ai
+description: Propose alternative hypotheses with specified emotional color.
+ram_usage: 500M
+gpu_usage: null
+group: candidate_annotators
+connector:
+  protocol: http
+  timeout: 15.0
+  url: http://emotional-bot-response:8050/respond_batch
+dialog_formatter: state_formatters.dp_formatters:bot_mood_emotion_formatter
+response_formatter: state_formatters.dp_formatters:simple_formatter_service
+previous_services: skills
+required_previous_services: skills
+state_manager_method: add_hypothesis_annotation_batch
+tags: null
+endpoint: respond_batch
```

Comment:
you have two endpoints in the component. Do you need respond endpoint? if you use it, then add a card for it. If not, simply remove from server.py

Reply:
removed `respond` endpoint